start

```diff
@@ -24,45 +24,26 @@ show_help() {
 
 while :; do
   case $1 in    
-    -d|--data-only)
-      sfd="false"
-      data="true"
-      receivers="true"
-      ;;
-    -r|--receivers-disabled)
-      receivers="false"
-      ;;  
```

Comment:
The `start` script has optional arguments that were carried over from the `ffc-sfd-core` repo, but considering we've been discussing not really needing the `customer-receiver` repos I've removed these arguments.